frontend/src/pages/PropertyDetails.jsx

```
@@ -108,34 +110,35 @@ function PropertyImageGallery({ images = [] }) {
+                    <button 
+                        className="relative w-[90%] h-[90%] max-w-[1200px] flex items-center justify-center bg-transparent border-0"
+                        onClick={(e) => e.stopPropagation()}
+                        onKeyDown={(e) => {
+                            if (e.key === "ArrowLeft" && images.length > 1 && currentImageIndex > 0) {
+                                handlePrev();
+                            } else if (e.key === "ArrowRight" && images.length > 1 && currentImageIndex < images.length - 1) {
+                                handleNext();
+                            }
+                        }}
+                        aria-label="Image container"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)